dist/package.json

```diff
@@ -20,7 +19,7 @@
     "brace": "^0.10.0",
     "chai": "^4.1.2",
     "es6-shim": "^0.35.3",
-    "grafana-sdk-mocks": "github:grafana/grafana-sdk-mocks",
+    "grafana-sdk-mocks": "github:briangann/grafana-sdk-mocks",
```

Comment:
I assume this is because we're blocked an PR from this fork? Can you file an issue for this project and link to that PR or change (if there is no PR back to origin).